templates/authorDepositForm.tpl

```diff
@@ -0,0 +1,100 @@
+{**
+ * plugins/generic/sword/templates/authorDepositForm.tpl
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Display list of deposit points.
+ *}
+{include file="common/header.tpl"}
+
+ <script type="text/javascript">
+	$(function() {ldelim}
+		$('#depositPointsTabs').pkpHandler('$.pkp.controllers.TabHandler');
+	{rdelim});
+</script>
+
+<div id="depositPointsTabs">
+	<ul>
+		<li><a href="#authorDepositPoints-tab">{translate key="plugins.generic.sword.displayName"}</a></li>
+	</ul>
+	<div id="authorDepositPoints-tab">
```

Comment:
It looks like this template was partially adapted from OJS 2.x style form markup; we're no longer using tables for form presentation.

Reply:
then this should be transformed into a grid?

Reply:
@asmecher : 

This is what the page looked like in OJS 2.x:

<img width="930" alt="screen shot 2018-05-08 at 6 31 31 pm" src="https://user-images.githubusercontent.com/4896270/39772905-5a75af5a-52ee-11e8-9858-7c93f95ba591.png">

And this is what  my OJS 3.x port looks like at the moment:

<img width="973" alt="screen shot 2018-05-08 at 6 31 49 pm" src="https://user-images.githubusercontent.com/4896270/39772953-7ac4b314-52ee-11e8-9895-14bac50d0fad.png">

What are tables replaced with in OJS 3.x? Point me to an example and I will make the change accordingly. thanks

Reply:
We now use `fbvFormSection` and `fbvFormArea` Smarty tags in place of where we used to use tables to organize elements. The user profile tabs probably provide the most concise examples of these.